sc/src/main/webapp/WEB-INF/jsp/tiles/views/home/experiment-list-latest.jsp

```diff
@@ -3,50 +3,47 @@
-                        <a href="experiments/${experimentInfo.experimentAccession}">
-                            <li class="flex-container">
-                                <div class="flex-item">
-                                    <c:choose>
-                                        <c:when test="${experimentInfo.experimentType.baseline}">
-                                            <span data-tooltip class="float-left baseline button-rd" title="Baseline experiment">B</span>
-                                        </c:when>
-                                        <c:otherwise>
-                                            <span data-tooltip class="float-left differential button-rd" title="Differential experiment">D</span>
-                                        </c:otherwise>
-                                    </c:choose>
-                                </div>
-                                <div class="flex-item" style="flex-grow:1;">
-                                    <small>${experimentInfo.lastUpdate}</small><br/>
-                                    ${experimentInfo.experimentDescription}
-                                    <i>${experimentInfo.species}</i>
-                                </div>
-                                <div class="flex-item">
-                                    <span class="button primary small">${experimentInfo.numberOfAssays}</span>
-                                    <span class="button small">Analysis result</span>
-                                    <span class="button icon icon-functional small" data-icon="="></span>
+        <div class="tabs-panel is-active" id="latest-experiments">
+            <ul class="clear">
+                <c:forEach items="${latestExperiments}" var="experimentInfo">
+                    <li>
+                        <div class="media-object stack-for-small middle">
+                            <div class="media-object-section">
+                                <c:choose>
+                                    <c:when test="${experimentInfo.experimentType.baseline}">
+                                        <span data-tooltip class="baseline button-rd" title="Baseline experiment">B</span>
+                                    </c:when>
+                                    <c:otherwise>
+                                        <span data-tooltip class="differential button-rd" title="Differential experiment">D</span>
+                                    </c:otherwise>
+                                </c:choose>
+                            </div>
+                            <div class="media-object-section middle">
+                                <div class="button-group">
+                                    <s:eval expression="T(java.text.NumberFormat).getInstance().format(experimentInfo.numberOfAssays)" var="fmtNumberOfAssays" />
+                                    <a class="button secondary no-action">${fmtNumberOfAssays}</a>
```

Comment:
This could be a `span` rather than a button (similar to the baseline `span`). However, it looks just like the `Analysis results` button so I imagine users would want to click on it (I certainly did!).

Reply:
I had some doubts here as well and the `no-action` class was the best I could think of. Maybe we want to add some tooltip or something. Do you have something specific in mind?